src/mlpack/methods/reinforcement_learning/policy/greedy_policy.hpp

```diff
@@ -41,13 +42,16 @@ class GreedyPolicy
    * @param annealInterval The steps during which the probability to explore
    *        will anneal.
    * @param minEpsilon Epsilon will never be less than this value.
+   * @param decayRate How much to change the model in response to the
+   *        estimated error each time the model weights are updated.
    */
   GreedyPolicy(const double initialEpsilon,
                const size_t annealInterval,
-               const double minEpsilon) :
+               const double minEpsilon,
+               const double decayRate) :
```

Comment:
To provide backward compatibility we should set a default value of one, that should also fix the build issue.